### PR TITLE
Changes and add style for the live editor.

### PR DESCRIPTION
**Pull request by fanxhe-pair**

### CODE REVIEW

**Comment by jtapia** [comment]
LGTM

File: public/assets/css/modules/documentation-menu.css [hunk 1]
```diff
@@ -21,13 +21,27 @@ input[class^="help-toggle"] + label {
...
   width: 20px;
 }
 
 .help-documentation {
-  left: 35px;
+  left: 60px;
+}
+
+.over:hover {
+  -webkit-transform: scale(1.);
```
Comment (ingilniero) [inline public/assets/css/modules/documentation-menu.css]: Is there a `2` missing here?

**Comment by ingilniero** [comment]
LGTM.

**Comment by joiggama** [comment]
:+1:

**Pull request merged**